### ISSUE TITLE
Lack of screenshots for assertion used directly in test

### ISSUE THREAD

**Issue by sobond** [serenity-bdd/serenity-core#2522]
Seems the Serenity doesn't support screenshots for assertion used directly in a test. When I tried to use `org.hamcrest.MatcherAssert.assertThat` or `org.junit.Assert.assertThat`, and call `assertThat` directly in Junit test (without rewriting to serenity step) I cannot attach the screenshot.
All `@Screenshots` annotations and `serenity.take.screenshots` properties refers to methods marked `@Step` annonation

<serenity-core.version>2.2.8</serenity-core.version>
<junit.version>4.12</junit.version>
<hamcrest-core.version>1.3</hamcrest-core.version>

![image](https://user-images.githubusercontent.com/42703604/127830171-95036d89-03c9-4f0f-93c8-0729c620b80f.png)
Code:
```
import static org.hamcrest.Matchers.is;
import static org.junit.Assert.assertThat;

@RunWith(SerenityRunner.class)
public class AssertionCheckStoryTests extends BaseTest {
        @Steps
        CommonSteps commonSteps;

    @Test
    @Screenshots(forEachAction = true)
    public void testDS() {
        commonSteps.opensUrl("https://www.google.com/");
        assertThat("One equals one", 1, is(1));
        commonSteps.opensUrl("http://www.thucydides.info/#/");
        assertThat("One equals two", 1, is(2));
    }

}
```

**Comment by wakaleo** [comment]
Serenity only does screenshots on annotated actions, so it can't do screenshots on plain assertions. You can use `Serenity.reportThat()` to get a screenshot, e.g.
```
        Serenity.reportThat("One equals two", () -> assertThat(1, is(2)));
```

**Comment by sobond** [comment]
Seems the `Serenity.reportThat()` method is not available in 2.2.8, but I got the idea. 
It's a pity that it cannot be handled by default, but thank you for your confirmation and quick response. 
I'll handle it a different way
